pkg/managed/services.go

```diff
@@ -22,33 +24,35 @@ func CreateServices(mbPair *types.ModelBindingPair, filteredConnections map[stri
 
 	glog.V(6).Infof("Creating/updating Service for VerrazzanoBinding %s", mbPair.Binding.Name)
 
-	// If binding is not System binding, skip creating Services
-	if mbPair.Binding.Name != constants.VmiSystemBindingName {
-		glog.V(6).Infof("Skip creating Services for VerrazzanoApplicationBinding %s", mbPair.Binding.Name)
-		return nil
-	}
-
 	// Construct services for each ManagedCluster
-	for clusterName := range mbPair.ManagedClusters {
+	for clusterName, mc := range mbPair.ManagedClusters {
 		managedClusterConnection := filteredConnections[clusterName]
 		managedClusterConnection.Lock.RLock()
 		defer managedClusterConnection.Lock.RUnlock()
 
+		var services []*corev1.Service
 		// Construct the set of expected Services
-		newServices := newServices(clusterName)
+		if mbPair.Binding.Name == constants.VmiSystemBindingName {
+			services = newServices(clusterName)
+		} else {
+			// Add services from genericComponents
+			for _, service := range mc.Services {
+				services = append(services, service)
+			}
+		}
 
 		// Create or update Services
-		for _, newService := range newServices {
+		for _, newService := range services {
```

Comment:
Since `newServices` was renamed, should `newService` be renamed to `service`?

Reply:
fixed